rails-snow-queen/app/views/quotes/new.html.erb

```diff
@@ -3,13 +3,29 @@
       <h1>Quote calculator</h1>
       <p>Get a free quote for a salting or snow clearing service</p>
 
-      <form action="" id="AddressSearch">
-         <label for="address">Search your location:</label><br>
-         <input id="address" type="textbox" placeholder="123 Street Rd, Port Moody" size="40">
-         <input id="submit" type="submit" value="Find" class="btn btn-light btn-sm">
-      </form>
+      <!-- ADDRESS SEARCH MODAL -->
+      <div class="modal fade" id="addressSubmitModal" tabindex="-1" role="dialog" aria-labelledby="addressSubmitModalLabel" aria-hidden="true" data-backdrop="static" data-keyboard="false">
+         <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+               <div class="modal-header">
+                  <h5 class="modal-title" id="addressSubmitModalLabel">Let's start looking for your address:</h5>
+               </div>
+               <form action="" id="AddressSearch">
+                  <div class="modal-body">
+                     <input id="address" type="textbox" placeholder="123 Street Rd, Port Moody" size="40">
```

Comment:
careful with your IDs... especially with such a generic name. You could very easily use the same ID on two elements and be unaware of it. Might want to change this to have a `name` attribute and you can get to it with `document.querySelector("#addressSearch [name='address']")`. Alternatively, with a `data` attribute

Reply:
done